<launch>

  <node pkg="ticker_publisher" type="ticker_publisher_node.py" name="ticker_publisher_btc_usd" output="screen">
    <param name="trade_pair" value="btc_usd"/>
  </node>

  <node pkg="ticker_publisher" type="ticker_publisher_node.py" name="ticker_publisher_ltc_usd" output="screen">
    <param name="trade_pair" value="ltc_usd"/>
  </node>

  <node pkg="ticker_publisher" type="ticker_publisher_node.py" name="ticker_publisher_ltc_btc" output="screen">
    <param name="trade_pair" value="ltc_btc"/>
  </node>
  
  <node pkg="macd_sell_signal" type="macd_sell_signal_node" name="macd_sell_signal_node" output="screen">
    <param name="trade_pair" value="btc_usd"/>
    <param name="short" value="12"/>
    <param name="long" value="25"/>
    <param name="sig" value="8"/>
    <param name="period" value="10"/>
    <param name="num_old_periods" value="15"/>
    <param name="spread_window" value="40"/>
    <param name="spread_value" value="1.26"/>   
  </node>

  <node pkg="sell_signal_filter" type="sell_signal_filter_node" name="sell_signal_filter_node" output="screen">	
  </node>


<!--
  <node pkg="plot_macd" type="plot_macd_node.py" name="plot_macd_btc_usd" output="screen">	
    <param name="macd_topic" value="macd_12_26_9_x60"/>
    <param name="ticker_topic" value="ticker_btc_usd"/>
    <param name="macd_array_service" value="macd_12_26_9_x60/macd_array"/>
  </node>
-->
  
</launch>